src/ui/component/walletAddress/view.jsx

```diff
@@ -1,5 +1,6 @@
 // @flow
 import React from 'react';
+import { REMOVE } from 'constants/icons';
```

Comment:
Can you changes these to 
`import * as ICONS from 'constants/icons'`

That makes it easier to search for when making changes

Reply:
Done!